src/navigations/root-stack.js

```diff
@@ -354,6 +355,11 @@ const AuthenticatedNavigator = () => {
           component={withSafeAreaView(withKeyboardWrapper(CreateCommunity))}
           options={{headerShown: false}}
         />
+        <AuthenticatedStack.Screen
+          name="CreateCommunityCustomize"
+          component={withSafeAreaView(CreateCommunityCustomize)}
+          options={{headerShown: false}}
+        />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `CreateCommunityCustomize` screen to the navigation stack seems fine. However, it's important to ensure that the necessary props are passed to this component when navigating to it. Also, make sure that the back navigation from this screen is handled correctly.
```diff
+         <AuthenticatedStack.Screen
+           name="CreateCommunityCustomize"
+           component={withSafeAreaView(CreateCommunityCustomize)}
+           options={{headerShown: false}}
+         />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->